lib/cartopy/tests/mpl/test_feature_artist.py

```diff
@@ -114,7 +114,7 @@ def styler(geom):
             return {'facecolor':'blue'}
 
     fig, ax = robinson_map()
-    ax.add_feature(feature, facecolor='red', styler=styler)
+    ax.add_feature(feature, facecolor='grey', styler=styler)
```

Comment:
Changing this to 'grey' helps distinguish between the failures 'styler was not applied' and 'styler was applied wrongly'.